@here/harp-mapview/lib/Utils.ts

```diff
@@ -201,6 +193,59 @@ export namespace MapViewUtils {
         );
     }
 
+    /**
+     * Calculate target (focus) point geo-coordinates for given camera.
+     * @see getTargetPositionFromCamera
+     *
+     * @param camera The camera looking on target point.
+     * @param projection The geo-projection used.
+     * @param elevation Optional elevation above (or below) sea level measured in world units.
+     */
+    export function getTargetCoordinatesFromCamera(
+        camera: THREE.Camera,
+        projection: Projection,
+        elevation?: number
+    ): GeoCoordinates | null {
+        // This function does almost the same as:
+        // rayCastGeoCoordinates(mapView, 0, 0)
+        // but in more gentle and performance wise manner
+        const targetWorldPos = getTargetPositionFromCamera(camera, projection, elevation);
+        if (targetWorldPos !== null) {
+            return projection.unprojectPoint(targetWorldPos);
+        }
+        return null;
+    }
+
+    /**
+     * Calculate target (focus) point world coordinates for given camera position and orientation.
+     * @param camera The camera looking on target point.
+     * @param projection The geo-projection used.
+     * @param elevation Optional elevation above (or below) sea level in world units.
+     */
+    export function getTargetPositionFromCamera(
```

Comment:
I would rather name the two methods:
- getGeoTargetFromCamera
- getWorldTargetFromCamera

Reply:
Fixed.